packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -357,7 +359,9 @@ export default class QuanticCaseClassification extends LightningElement {
     if (Number(value) >= 0) {
       this._maxSuggestions = Number(value);
     } else {
-      console.warn('Please enter a valid number of maximum suggestions');
+      console.warn(
+        'Invalid number of maximum suggestions introduced, the default value is being used'
```

Comment:
The part `the default value is being used` can be misleading as the value is not updated with the default, it is simply not updated when the specified value is smaller than zero.

I would suggest this

    `"${value}" is an invalid maximum number of suggestions. A positive integer was expected. "${this._maxSuggestions}" will be used instead.`

Reply:
I think defaulting back to another value might be too subtle and just unnecessary. This error should only happen when the developer is setting up the page so I think letting the component fail gently while rendering a red error message should be enough to allow them to fix it.